app/src/main/kotlin/com/wire/android/ui/home/settings/appsettings/networkSettings/NetworkSettingsScreen.kt

```diff
@@ -84,7 +85,7 @@ fun NetworkSettingsScreenContent(
 @Composable
 private fun getSwitchState(isWebSocketEnabled: Boolean, setWebSocketState: (Boolean) -> Unit): SwitchState {
     val context = LocalContext.current
-    return if (context.isGoogleServicesAvailable()) {
+    return if (BuildConfig.WEBSOCKET_ENABLED_BY_DEFAULT || context.isGoogleServicesAvailable()) {
```

Comment:
this line is duplicated can we extract it to its own usecase ?

Reply:
I'll try to create something for future reusage, but the check is not the same actually